core/ui/AdvancedSearch/Filter.tid

```diff
@@ -3,11 +3,19 @@ tags: $:/tags/AdvancedSearch
 caption: {{$:/language/Search/Filter/Caption}}
 
 \define lingo-base() $:/language/Search/
-\define set-next-input-tab(beforeafter:"after")
-<$macrocall $name="change-input-tab"
+\define set-next-input-tab()
```

Comment:
macros should be replaced with procedures

Reply:
done